test/cases/spirv_mergable_pointers.zig

```diff
@@ -14,3 +14,4 @@ export fn a() void {
 // output_mode=Obj
 // backend=stage2
 // target=spirv64-vulkan
+// emit_bin=false
```

Comment:
FYI @Snektron 1d1ca8485454a4b14f20376aff532bfe7c959155

Reply:
Should be fine